developments_build/sql/_geo.sql

```diff
@@ -355,18 +337,19 @@ correction into the corrections_applied table.
 Append details of distance and spatial join checks to reason.
 */
 INSERT INTO corrections_applied 
+SELECT
     job_number, 
     field,
     (CASE
         WHEN field = 'latitude' THEN current_latitude 
         WHEN field = 'longitude' THEN current_longitude
     END) as current_value,
-    old_value,
-    new_value,
+    old_geom,
```

Comment:
We still want the corrections inserted into corrections_applied and corrections_not_applied to be the original latitude longitude corrections.